src/pages/gallery.js

```diff
@@ -263,15 +266,15 @@ export default function Gallery() {
                 <span>Could not load gallery, check your connection and try again</span>
               </div>
 
-            ) : eventStatus === 'succeeded' ? (
+            )}
+            {indexFetchStatus === FETCH_INDEX_PAGE_INFO_STATUS.SUCCEEDED && (
               (search?.length === 0) ? <div className='failed-search'>
                 <img src={FailedSearch} alt='Failed search'/>
                 <h3>No results for that search :(</h3>
               </div> :
               <Cards events={(search?.length) ? search : items} length={search?.length || length} />
-            ) : (
-              <Loader/>
             )}
+            {(indexFetchStatus === FETCH_INDEX_PAGE_INFO_STATUS.IDLE || indexFetchStatus === FETCH_INDEX_PAGE_INFO_STATUS.IDLE) && <Loader/>}
```

Comment:
duplicate condition? `indexFetchStatus === FETCH_INDEX_PAGE_INFO_STATUS.IDLE`
```suggestion
            {(indexFetchStatus === FETCH_INDEX_PAGE_INFO_STATUS.IDLE) && <Loader/>}
```

Reply:
buen ojo! ahi va IDLE y LOADING